File: sudoku-common/common-core/src/main/java/com/zumin/sudoku/common/core/constant/AuthRedisKey.java
package com.zumin.sudoku.common.core.constant;

/**
 * Redis键
 */
public interface AuthRedisKey {

  /**
   * 黑名单token前缀
   */
  String TOKEN_BLACKLIST_PREFIX = "auth:token:blacklist:";

  /**
   * 验证码前缀
   */
  String CAPTCHA_PREFIX = "auth:captcha:";
}

File: sudoku-common/common-core/src/main/java/com/zumin/sudoku/common/core/auth/AuthRedisKey.kt
package com.zumin.sudoku.common.core.auth


/**
 * 黑名单token前缀
 */
const val TOKEN_BLACKLIST_PREFIX = "auth:token:blacklist:"

/**
 * 验证码前缀
 */
const val CAPTCHA_PREFIX = "auth:captcha:"

/**
 * Redis缓存权限规则key
 */
const val RESOURCE_ROLES_KEY = "auth:resource:roles"